doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -44,3 +46,58 @@ The following table illustrates ABI compatibility between different versions of
 
 ABI compatibility ensures that the Secure Domain and System Controller firmware binaries do not need to be recompiled each time the Application, Radio binaries, or both are recompiled, as long as they are based on a compatible |NCS| version.
 Additionally, maintaining ABI compatibility allows the nRF54H20 SoC binaries components to work together without recompilation when updating to newer |NCS| versions.
+
+nRF54H20 SoC Binaries v0.8.0 changelog
+**************************************
+
+The following sections provide detailed lists of changes by component.
+
+Secure Domain Firmware (SDFW) v.9.0.0
+=====================================
+
+Added
+-----
+
+* Purge protection can be enabled over ADAC.
+* Clock control is enabled in SDFW.
```

Comment:
I think these and the last one belong to the updated section.

Reply:
These seems to be like additional features, and this is a changelog that comes from the SDFW one. It would diverge from that one if items were to be moved.